src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/LoongArch64/LA_LASX.cs

```diff
@@ -0,0 +1,2269 @@
+        ///   LASX: XVFMAX.d Xd.4D, Xj.4D, Xk.4D
+        /// </summary>
+        public static Vector256<double> Max(Vector256<double> left, Vector256<double> right) => Max(left, right);
+
+        /// <summary>
+        /// int8x32_t xvmin_b_s8 (int8x32_t a, int8x32_t b)
+        ///   LASX: XVMIN.B Xd.32B, Xj.32B, Xk.32B
+        /// </summary>
+        public static Vector256<sbyte> Min(Vector256<sbyte> left, Vector256<sbyte> right) => Min(left, right);
+
+        /// <summary>
+        /// uint8x32_t xvmin_bu_u8 (uint8x32_t a, uint8x32_t b)
+        ///   LASX: XVMIN.BU Xd.32B, Xj.32B, Xk.32B
+        /// </summary>
+        public static Vector256<byte> Min(Vector256<byte> left, Vector256<byte> right) => Min(left, right);
+
+        /// <summary>
+        /// int16x16_t xvmin_h_s16 (int16x16_t a, int16x16_t b)
+        ///   LASX: XVMIN.H Xd.16H, Xj.16H, Xk.16H
+        /// </summary>
+        public static Vector256<short> Min(Vector256<short> left, Vector256<short> right) => Min(left, right);
+
+        /// <summary>
+        /// uint16x16_t xvmin_hu_u16 (uint16x16_t a, uint16x16_t b)
+        ///   LASX: XVMIN.HU Xd.16H, Xj.16H, Xk.16H
+        /// </summary>
+        public static Vector256<ushort> Min(Vector256<ushort> left, Vector256<ushort> right) => Min(left, right);
+
+        /// <summary>
+        /// int32x8_t xvmin_w_s32 (int32x8_t a, int32x8_t b)
+        ///   LASX: XVMIN.W Xd.8S, Xj.8S, Xk.8S
+        /// </summary>
+        public static Vector256<int> Min(Vector256<int> left, Vector256<int> right) => Min(left, right);
+
+        /// <summary>
+        /// uint32x8_t xvmin_wu_u32 (uint32x8_t a, uint32x8_t b)
+        ///   LASX: XVMIN.WU Xd.8S, Xj.8S, Xk.8S
+        /// </summary>
+        public static Vector256<uint> Min(Vector256<uint> left, Vector256<uint> right) => Min(left, right);
+
+        /// <summary>
+        /// int64x4_t xvmin_d_s64 (int64x4_t a, int64x4_t b)
+        ///   LASX: XVMIN.D Xd.8S, Xj.8S, Xk.8S
+        /// </summary>
+        public static Vector256<long> Min(Vector256<long> left, Vector256<long> right) => Min(left, right);
+
+        /// <summary>
+        /// uint64x4_t xvmin_du_u64 (uint64x4_t a, uint64x4_t b)
+        ///   LASX: XVMIN.DU Xd.8S, Xj.8S, Xk.8S
+        /// </summary>
+        public static Vector256<ulong> Min(Vector256<ulong> left, Vector256<ulong> right) => Min(left, right);
+
+        /// <summary>
+        /// float32x8_t xvfmin_s_f32 (float32x8_t a, float32x8_t b)
+        ///   LASX: XVFMIN.S Xd.8S, Xj.8S, Xk.8S
+        /// </summary>
+        public static Vector256<float> Min(Vector256<float> left, Vector256<float> right) => Min(left, right);
+
+        /// <summary>
+        /// float64x4_t xvfmin_d_f64 (float64x4_t a, float64x4_t b)
+        ///   LASX: XVFMIN.D Xd.4D, Xj.4D, Xk.4D
+        /// </summary>
+        public static Vector256<double> Min(Vector256<double> left, Vector256<double> right) => Min(left, right);
```

Comment:
What's the semantics around `NaN` and `-0` handling on LoongArch?

Reply:
The float operation is implemented within the IEEE-754-2008, here is MinNum(x,y).